test/net/sourceforge/zmanim/hebrewcalendar/UT_GregorianDateNavigation.java

```diff
@@ -28,21 +29,21 @@ public void gregorianForwardMonthToMonth() {
 		Assert.assertEquals(26, hebrewDate.getJewishDayOfMonth());
 
 		hebrewDate.forward();
-		Assert.assertEquals(2, hebrewDate.getGregorianMonth());
+		Assert.assertEquals(Calendar.FEBRUARY, hebrewDate.getGregorianMonth());
```

Comment:
FWIW, this is a good example of the confusion... We now require devs to know when to use the Calendar constants, and when to use the MONTH_ constants...